examples/qubit_tuneup/CPhase_tuneup.py

```diff
@@ -31,7 +31,7 @@
               DataT.RO_acq_weight_function_I()],
     nr_averages=AncT.RO_acq_averages(),
     integration_length=AncT.RO_acq_integration_length(),
-    cross_talk_suppression=True)
+    result_logging_mode='normalized')
```

Comment:
as discussed with @AdriaanRol: change 'normalized' to 'lintrans' as it is a general linear transformation, not just normalization.